### DISCUSSION TITLE
Images deleted from external library storage still showing up in the app

### DISCUSSION THREAD

**Discussion by kikendo** [immich-app/immich#9936] [Q&A] [answered]
### The bug

I deleted a bunch of files from the external library storage I set up, and even after full library re-scan, I still see the same images I deleted on my timeline.

### The OS that Immich Server is running on

Debian 11

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

v1.105.0 build.157

### Platform with the issue

- [X] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment, marked as answer]
Did you run the remove offline files operation on the library?

**Reply by kikendo** [reply]
Mark this issue as "user idiot error". 
I never saw that option. Now it seems to be working. 
Thanks and sorry!

**Comment by jrasm91** [comment]
Your volume for external libraries is mounted as read only so the file won't get deleted in disk and it will be picked up on the next scan.

**Reply by kikendo** [reply]
I deleted it from the volume in the host system, not via Immich
